web/styles/app_components.css

```diff
@@ -542,6 +542,10 @@ input.settings_text_input {
     }
 }
 
+.desktop-setting-notifications {
+    margin-bottom: 10px;
+}
```

Comment:
I feel like ideally the components would come with a reasonable bottom margin. But this is probably OK for now.